File: exercicios/Python/2140/2140.py
n = [4,7,12,15,22,25,30,40,52,55,60,70,100,102,105,110,120,150,200]

while 1:
    try:
        v = input()
        v = v.split(' ')
        t = int(v[0])
        p = int(v[1])

        troco = p - t
        imp = 0
        pos = 0
        for i in range(len(n)):
            if (troco == n[i]):
                
                v = True
                 
        if v == True:
            print('possible')
        else:
            print('impossible')
    
    except:
        if p == 0 or t == 0:
            break

File: exercicios/JavaScript/2140/2140.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

while(true){

    var troco = 0;

    var [n,m] = lines.shift().split(' ').map(Number);

    if(n == 0 && m == 0){
        break;
    }
    troco = m - n
    if(troco == 7 || troco == 12 || troco == 22 || troco == 52 || troco == 102 || troco == 15 || troco == 25 || troco == 55 || troco == 105 || troco == 30 || troco == 60 || troco == 110 || troco == 70 || troco == 120 || troco == 150){
    console.log("possible")}
    else{
    console.log("impossible")}   
}
